editor/plugins/org.fusesource.ide.projecttemplates/plugin.xml

```diff
@@ -186,7 +186,7 @@
       
       	<project-facet-version
             facet="jst.camel"
-            version="2.15.1.redhat-621084">
+            version="1.0">
```

Comment:
shouldn't we use a 3.0? like that we are higher than the previous 2.x provided one.